packages/theme/src/theme/_forms.switch.scss

```diff
@@ -0,0 +1,83 @@
+.switch {
+	input[type='checkbox'],
+	.checkbox input[type='checkbox'],
+	&.checkbox input[type='checkbox'] {
+		position: absolute;
+		clip: rect(1px, 1px, 1px, 1px);
+		padding: 0;
+		border: none;
+		height: 1px;
+		width: 1px;
+		overflow: hidden;
+
+		+ * {
+			padding-left: 2rem;
+		}
+
+		+ *:before,
+		+ *:after,
+		&:checked + *:before,
+		&:checked + *:after {
+			position: absolute;
+			content: '';
+			top: 0;
+			left: 0;
+			transition: background 0.2s ease, transform 0.2s ease;
+			border-radius: 10rem;
+			border-color: transparent;
```

Comment:
This seems to be only on `:after` (on `:before` we have no border)

Reply:
thanks, I will fix it